tests/subsys/bluetooth/mesh/sensor_subsys/testcase.yaml

```diff
@@ -1,6 +1,6 @@
 tests:
   bluetooth.mesh.sensor_subsys:
-    platform_allow: native_posix
+    platform_allow: native_posix qemu_cortex_m3
     tags: bluetooth mesh models sensor
```

Comment:
Should we change the tags to "bluetooth ci_build" at the same time for sensor and silvair.

Reply:
Don't know. Does it impact anything? Looks like it works well with current tags.

Reply:
@alxelax: Relevant conversation: https://github.com/nrfconnect/sdk-nrf/pull/5057